### PR TITLE
Fea/table row strategy provider list view

### PR DESCRIPTION
**Pull request by digvijay-gurav**

### CODE REVIEW

File: strategy-providers-list.html [hunk 1]
```diff
@@ -306,14 +306,14 @@ <h2 class="font-bold my-2">Strategy Providers</h2>
...
                         <tr>
-                          <th>PROVIDER</th>
-                          <th>RETURN</th>
-                          <th>RISK</th>
-                          <th>TRENDLINE</th>
-                          <th>COPIER</th>
-                          <th>RATING</th>
-                          <th>COPY</th>
-                          <th>WATCH</th>
+                          <th class="table-header-cell">PROVIDER</th>
```
Comment (neerajdembla212) [inline strategy-providers-list.html:309]: its better to add this style using something like `table > head th` so we don't have to use this class on every cell, which looks more clean

File: strategy-providers-list.html [hunk 2]
```diff
@@ -325,13 +325,12 @@ <h2 class="font-bold my-2">Strategy Providers</h2>
...
                               />
-                              <div class="d-flex flex-column ml-3">
+                              <div class="d-flex flex-column ml-2">
                                 <span class="font-bold font-size-12"
                                   >SureWin888</span
                                 >
                                 <span class="text-light-black font-size-12"
-                                  >Chun Eng Lee</span
-                                >
+                                  >Chun Eng Lee <i class="ml-1 fa fa-flag"></i></span>
```
Comment (neerajdembla212) [inline strategy-providers-list.html:333]: we need country flags as per design. do we have different country flags in theme ?

File: strategy-providers-list.html [hunk 3]
```diff
@@ -344,7 +343,7 @@ <h2 class="font-bold my-2">Strategy Providers</h2>
                           <td class="pt-pb-20">
                             <canvas class="triangle"></canvas>
                           </td>
-                          <td class="canvas-max-width-180 pt-pb-20">
+                          <td class="canvas-max-width-trendline pt-pb-20">
```
Comment (neerajdembla212) [inline strategy-providers-list.html:346]: This comment is for last part of this row where we have favourites icon. According to design we need something like icon button on list page. you may try adding following icon button from theme : 

Screenshot 2021-07-09 at 10 52 14 AM

**Review by neerajdembla212 — CHANGES_REQUESTED** [review]

File: strategy-providers-list.html (hunk 1 above)
Comment (digvijay-gurav) [reply strategy-providers-list.html]: done

File: strategy-providers-list.html (hunk 2 above)
Comment (digvijay-gurav) [reply strategy-providers-list.html:333]: I did check the font awesome library and the theme as well but didn't find the maps we want.

File: strategy-providers-list.html (hunk 3 above)
Comment (digvijay-gurav) [reply strategy-providers-list.html:346]: Added a button and wrapped the icon within it

**Pull request merged**